### PR TITLE
security: postmortem for CVE-2019-990[01]

### PR DESCRIPTION
**Pull request by htuch**

### CODE REVIEW

**Comment by htuch** [comment]
@lizan @mattklein123 @destijl @liggitt @duderino @silentdai @PiotrSikora @jplevyak @fpesce @yangminzhu @liminw  FYI

**Comment by htuch** [comment]
I don't think we had any additional changes to this document that came up yesterday, but if I missed something please chime in. Otherwise, maybe merge in a day or 2 if there's no further feedback?

File: security/postmortems/cve-2019-9900.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+* Fix patches were available within 2 weeks of vulnerability disclosure. The
+  changes were localized and relatively clean.
+
+### What went wrong
+
+* The Envoy private distributor list was initially almost empty. We sent out an
+  e-mail to remind distributors to sign up on 2019-03-14 and the list is now O(10).
+
+* The security impact of https://github.com/envoyproxy/envoy/issues/6008
```
Comment (PiotrSikora) [inline security/postmortems/cve-2019-9900.md]: Maybe add a comment about the fact that https://github.com/envoyproxy/envoy/issues/2956 was opened a year earlier and also wasn't caught as a potential security issue? This is mentioned in the timeline, but that's a section that's usually skipped.

File: security/postmortems/cve-2019-9900.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+  potential effects of this bugs. While it was unclear whether there would be
+  impact beyond some narrow circumstances, agreement was reached to start the
+  security release process. Analysis began to determine the extent of impact on
+  HTTP/2 and Envoy's code base was audited.
+
+2019-03-11:
+* [CVE-2019-9901] https://github.com/envoyproxy/envoy/pull/6258 was opened to address
+  https://github.com/envoyproxy/envoy/issues/6008.
+
+
```
Comment (PiotrSikora) [inline security/postmortems/cve-2019-9900.md]: Nit: 2 empty lines.

File: security/postmortems/cve-2019-9900.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+  severity in the Istio setup (in particular with RBAC and Mixer in play).
+
+2019-03-14:
+* [CVE-2019-9900] Finding were presented to envoy-security@. A fix plan was
+  agreed upon and a candidate fix PR was shared with the team by e-mail. At this
+  point, no private fix repository existed.
+* [CVE-2019-9901] The Istio fix leads initiated private work on a fix patch.
+  Since it was likely that this would land within the 1.9.1 release
+  window for CVE-2019-9900, CVE-2019-9901 was also scheduled for the release.
+* [Announcement](https://groups.google.com/forum/#!topic/envoy-announce/dEOLqAiaSUI) sent to remind distributors to join
```
Comment (PiotrSikora) [inline security/postmortems/cve-2019-9900.md]: Nit: line too long.

File: security/postmortems/cve-2019-9900.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+  cncf-envoy-distributors-announce@lists.cncf.io.
+
+2019-03-20:
+* CVEs were requested from MITRE for both issues.
+* Draft fix PRs for CVE-2019-9900 and CVE-2019-9901 were shared on private Envoy
+  security repository. Reviews and further development occurred over the
+  following week.
+
+2019-03-22:
+* 11:20 1.9.1 security release for the two vulnerabilities was [announced](https://groups.google.com/d/msg/envoy-announce/6fwGB2TxB74/dKeURAdfAgAJ).
```
Comment (PiotrSikora) [inline security/postmortems/cve-2019-9900.md]: Nit: line too long.

**Pull request merged**